frontend/src/components/NotificationsMunicipality.vue

```diff
@@ -1,61 +1,141 @@
 <template>
```

Comment:
A LOT of duplicated code between NotificationsCounty.vue and NotificationsMunicipality.vue